editor.planx.uk/src/ui/InputRow.tsx

```diff
@@ -1,66 +1,38 @@
-import { makeStyles } from "@mui/styles";
-import classNames from "classnames";
+import { BoxProps } from "@material-ui/core";
+import Box from "@mui/material/Box";
+import { styled } from "@mui/material/styles";
 import React from "react";
 
-import type { Theme } from "../theme";
-
-const useClasses = makeStyles((theme: Theme) => ({
-  inputRow: {
-    display: "flex",
-    width: "100%",
-    "&:not(:last-child)": {
-      marginBottom: 2,
+const Root = styled(Box, {
+  shouldForwardProp: (prop) => prop !== "childRow",
+})<BoxProps & { childRow?: boolean }>(({ childRow }) => ({
+  display: "flex",
+  width: "100%",
+  "&:not(:last-child)": {
+    marginBottom: 2,
+  },
+  "& > *": {
+    flexGrow: 1,
+    marginLeft: 1,
+    marginRight: 1,
+    "&:first-child": {
+      marginLeft: 0,
     },
-    "& > *": {
-      flexGrow: 1,
-      marginLeft: 1,
-      marginRight: 1,
-      "&:first-child": {
-        marginLeft: 0,
-      },
-      "&:last-child": {
-        marginRight: 0,
-      },
+    "&:last-child": {
+      marginRight: 0,
     },
   },
-  childRow: {
+  ...(childRow && {
     paddingLeft: 52,
     position: "relative",
-  },
-  rowIcon: {
-    color: theme.palette.text.secondary,
-    opacity: 0.5,
-    position: "absolute",
-    left: 13,
-    top: 13,
-    "& + *": {
-      marginLeft: 0,
-    },
-  },
+  }),
 }));
 
 interface Props {
-  RowIcon?: any;
```

Comment:
This prop is never used throughout the application so I've removed this and any associated references.